### ISSUE TITLE
Face tagging/recognition available?

### ISSUE THREAD

**Issue by Suncatcher** [Piwigo/Piwigo#1159]
Does Piwigo have full-featured face-tagging functionality [like Picasa had](https://lifehacker.com/picasa-3-5-organizes-your-photos-with-facial-recognitio-5365267)? Either in plugins or in basic functionality?

**Comment by sridharb1** [comment]
I continue to use Picasa to organize my photographs while "exporting" selected pictures to my Piwigo website to share with family.

I extensively use Picasa Albums and face tags. There is limited support in piwigo through a plugin called mugshot. I have some issues with this plugin (need to refresh the page manually for the tags to show), but the development is dead and the author is unreachable. I have written software for personal use to export picasa faces to the mugshot format. It is working fine for now.

TLDR, no and plugin support is very basic.

**Comment by Suncatcher** [comment]
Thanks for info, man. It is very helpful.

**Comment by sridharb1** [comment]
Please let me know what you end up using for faces. If mugshot, and you improve on it, would like to know.

**Comment by Suncatcher** [comment]
Unlikely I will use mugshot as it [can](https://github.com/cccraig/MugShot) only face-tagging but not recognition, and I need more full-featured solution.
